### DISCUSSION TITLE
Adding colors to specific headings

### DISCUSSION THREAD

**Discussion by mmpadilla15** [thegetty/quire#456] [Get Help] [answered]
I'm creating a page that will have headings in a specific color depending on the text that follows. For example:

###Blue
My favorite color is blue because the sky is blue

Also, the ocean is sorta blue.

###Green
Grass is green and so are grasshoppers.

###Blue
I like Picasso's Blue period.

I was able to change the color of the level three heading by adding a span tag around it. For example: 
`<span style="color: red;">Red</span>` But I can't seem to change the color when I insert the hex number of the color I want to use.

Any advice?

**Comment by geealbers** [comment, marked as answer]
Hi @mmpadilla15, just to clarify, it's not working when you do something like `<span style="color: #f55a42;">Hex color</span>`? Could you share with us the specific markup you're using?

**Reply by mmpadilla15** [reply]
It looks like it's working now. Not sure what I was doing wrong, but when I copied and pasted your code and replaced with my hex number, it works.
